### ISSUE TITLE
Data from Poller is rewriting DOM constantly (causing issues)

### ISSUE THREAD

**Issue by jjansenvr** [frankframework/frankframework#6525]
**❗ please do not add sensitive information in issues, you can provide extra information via email using issue number as reference ❗**

**Describe the issue**
While working with the GUI you want to be able to copy f.e. a queue name from the adapter page but the only way to do this is select the text hold the mouse and ctrl-C simultaneously

I would like to be able to copy texts anywhere from the Gui

**Reporter**
jjansenvr

**Comment by Matthbo** [comment]
This seems to happen on the status page, each time the configuration poller callback has run and on the expanded adapters where some adapter information is updated.

Strangely this doesn't seem to occur on adapters that don't have updated info/stats.

_note: this is before improvements of AppComponent, AppService & Poller, this should be tested again after_

**Comment by Matthbo** [comment]
Maybe diving parts of the status page into nested components will help debug this behaviour more

**Comment by mhdirkse** [comment]
I wrote an automated test to check that my adapter does not have warnings. I have an adapter that definitely has warnings:

![image](https://github.com/frankframework/frankframework/assets/6784732/a92f6714-f923-4043-8c24-0931fdb2d5b7)

The DOM elements of the warnings have attribute `data-cy-status-alert`. My Cypress test checks that there are no DOM elements with attribute `data-cy-status-alert`. The test should fail but it succeeds.

When I look at the Cypress window I see a stream of HTTP requests:

![image](https://github.com/frankframework/frankframework/assets/6784732/0a4c1386-8702-4cff-a347-8fa3a2e58b28)

It looks like the list of alerts is refreshed after each of these requests. During a refresh the status warnings may vanish, causing the test to succeed unjustified.

**Comment by mhdirkse** [comment]
My previous comment is wrong. When I change my test to have: `cy.get('.alert-warning', {timeout: 10000}).should('have.length', 0)`, then it behaves as I want. Probably I put attribute `data-cy-status-alert` on the wrong element in the HTML source code.

**Comment by mhdirkse** [comment]
I updated my test not to depend on some data-cy. Now I think this issue is indeed relevant for my Cypress test:

![image](https://github.com/frankframework/frankframework/assets/6784732/aa6fb5a8-d4db-4c22-a7a9-24642d1f74b2)

**Comment by nielsm5** [comment]
Are you sure you're using the correct selector? I believe your problem is unrelated...

**Comment by mhdirkse** [comment]
In my cypress test I made a workaround that checks multiple times whether there are no warnings. I will remove that workaround and check whether that makes the test flaky or incorrect.

**Comment by mhdirkse** [comment]
@nielsm5, I think you are right that the issue with my Cypress test is unrelated to the present issue. To check this, I first simplified my test to check only once that there are no warnings. Then my test erroneously passed - my config had an explicit attribute that repeated its default value. Then I added a wait of 5 seconds after my cy.visit() and before checking that there are no warnings. Then my test failed again as expected. It seems that not all elements of the adapter status page appear at the same time.
